### ISSUE TITLE
Glimpse 0.1.2 does not start anymore

### ISSUE THREAD

**Issue by JakubValtar** [glimpse-editor/Glimpse#442]
Glimpse was working fine yesterday, but I can't get it to launch today. Nothing happens after clicking on the shortcut or running Glimpse.exe from the command line. There is a spinning wheel for a few seconds next to my cursor, then the wheel disappears.

Before this happened I associated xcf files with Glimpse. Not sure if that is the cause or not.

Since then I removed the association, uninstalled Glimpse, removed the app folders from AppData\Local (there were no crash logs in there) and AppData\Remote, and reinstalled. Didn't help.

### To Reproduce

1. Double click on the Glimpse shortcut
2. Nothing happens

Command line:

1. Run `.\Glimpse.exe` from `C:\Program Files (x86)\Glimpse Image Editor\Glimpse 0.1.2\bin` 
 or `.\bin\Glimpse.exe` from `C:\Program Files (x86)\Glimpse Image Editor\Glimpse 0.1.2`
 or `.\Glimpse.exe --verbose --console-messages` from `C:\Program Files (x86)\Glimpse Image Editor\Glimpse 0.1.2\bin`
 or `.\bin\Glimpse.exe --verbose --console-messages` from `C:\Program Files (x86)\Glimpse Image Editor\Glimpse 0.1.2`
2. The command terminates immediately and nothing happens.

### Expected behavior

Splash screen appears and the editor starts loading.

### Desktop
OS: Windows
Version: Windows 10 (1909)

### Verbose Logs
Post logs from running the application with the `--verbose --console-messages` flags

When I run the app with these flags, the command does not produce any output and terminates. No windows appear.

**Comment by TrechNex** [comment]
You can check if a process is running in the Task Manager (Ctrl+Shift+Escape will pop this up).

Sounds obvious, but have you tried restarting your machine between uninstall/reinstall? If that doesn't kill off any intermittent problems, you might find that there's a rogue Windows Update of some description messing with you.

The `Glimpse.exe` won't run without setting the working directory as it needs to be able to see the `.DLL` files it is included with. You can try this in Powershell with the following command:
```
Start-Process -FilePath Glimpse.exe -WorkingDirectory "C:\Program Files (x86)\Glimpse Image Editor\Glimpse 0.1.2\bin"
```

If that works, then it might be worth checking if the working directory is correct in the shortcut that is saved on your desktop.

**Comment by TrechNex** [comment]
The other thing worth checking would be if your anti-virus program has quarantined any files recently. If you discover a file from the Glimpse installation directory then restoring it can also fix problems like this.

**Comment by TrechNex** [comment]
Tagging this to the 0.2.0 milestone.

For background the next stable release is planned for next week. The Windows installers for 0.2.0 have been rewritten completely from scratch, and should be more reliable because they treat every file as a component that needs to be verified (currently we treat every folder as a component, so the installer just checks it's there and at least one file is in it!)

**Comment by JakubValtar** [comment]
I checked Windows Update and it installed Nahimic. It is an audio "enhancement" software which injects DLLs into running programs to do the sound processing. Problem is that some programs don't like it and won't start. I have it disabled, but it gets enabled again after update.

I disabled the Nahimic service which does the injection and Glimpse works again! (Until another update enabled the service.)

Thanks for the help and sorry for the bug report.
